src/store/modules/keymap.js

```diff
@@ -275,6 +275,23 @@ const mutations = {
     }
     state.layer = newLayer;
   },
+  swapLayers(state, targetLayer) {
+    if (parseInt(targetLayer) === state.layer) {
+      return;
+    }
+
+    if (state.keymap[targetLayer] === undefined) {
+      mutations.initLayer(state, targetLayer);
+    }
+
+    [state.keymap[targetLayer], state.keymap[state.layer]] = [
```

Comment:
not sure that's a good way to do it. You'd probably want to use Vue.set to make sure the reactivity system is kept in sync